tests.sh

```diff
@@ -223,6 +223,11 @@ build_objectivec_osx() {
       --core-only --skip-xcode-ios
 }
 
+build_objectivec_cocoapods_integration() {
+  gem install cocoapods -v '1.0.1' --no-ri --no-rdoc
```

Comment:
Do we have to put a version or gan we generically just trigger an update so it always uses what is current?